app.js

```diff
@@ -31,9 +31,36 @@
 // 74 - Soft Tek Front
 
 let drums = null;
+let buttons = null;
+const config = {}
+const DEFAULT_SOUNDFONT =  'Doumbek-Faisal';
+const DEFAULT_INSTRUMENT = 'doumbek';
+const DEFAULT_BUTTONS = {
+  "b1": 64,
+  "b2": 65,
+  "b3": 66,
+  "b4": 67,
+  "b5": 68,
+  "b6": 69
+};
+
+async function loadSoundFonts(soundfont) {
+  const soundFonts = await fetch('/soundfonts.json')
+    .then(data => data.json())
+    .then(json => json)
+  return soundFonts[soundfont]
+}
+
+async function loadConfig(soundfont, instrument) {
```

Comment:
I think the functions `loadSoundFonts` and `loadConfig` should be a single one that just loads the `soundfont.json` file and places it in `config`. We can then call this function when the script loads, and any subsequent function just reads from `config`.